thunder/tests/test_core.py

```diff
@@ -818,62 +769,42 @@ def foo(a, b):
 
     # Tensor x tensor
     result = cfoo(a, b)
-    assert cache_misses(cfoo) == 1
-    assert cache_hits(cfoo) == 0
```

Comment:
Re https://github.com/Lightning-AI/lightning-thunder/pull/1410#issuecomment-2594603044
I think t-vi refers to these removals. Why are these asserts removed?